SwiftfulSourceControl/SettingsView.swift

```diff
@@ -8,8 +8,9 @@
 import SwiftUI
 
 struct SettingsView: View {
+    @State private var doSometThing : Bool = false
```

Comment:
Hey this works but we don't use nameing convention can you rename this to "isPremium" ?